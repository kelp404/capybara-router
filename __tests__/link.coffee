React = require 'react'
renderer = require 'react-test-renderer'
router = require '../'


test 'Link component render', ->
  component = renderer.create do ->
    <router.Link href="https://github.com">GitHub</router.Link>
  tree = component.toJSON()
  expect(tree).toMatchSnapshot()
