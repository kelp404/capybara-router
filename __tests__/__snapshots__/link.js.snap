// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component render 1`] = `
<a
  href="https://github.com"
  onClick={[Function]}
>
  GitHub
</a>
`;
